encode_test.go

```diff
@@ -165,6 +166,26 @@ var marshalTests = []struct {
 		},
 		expect: loadTestData("marshal-key-escape.toml"),
 	},
+	// empty interface with no type knowledge at compile time:
+	{
+		v: func() interface{} {
+			var (
+				v   = new(interface{})
+				err error
+			)
+
+			err = json.Unmarshal(
+				[]byte("{\"foo\":\"bar\"}"),
+				v,
+			)
+			if err != nil {
+				panic(err)
+			}
```

Comment:
You don't need encoding/json to make this value. Please use

```
v: func () interface{} {
    var v interface{} = map[string]interface{}{"foo": "bar"}
    return &v
}()
```

Reply:
Yeah, you are right, fixed that